policyengine_us/parameters/gov/states/ne/tax/income/agi/subtractions/military_retirement/fraction.yaml

```diff
@@ -0,0 +1,13 @@
+description: Fraction of military retirement benefits allowed as NE AGI subtraction.
+values:
+  2022-01-01: 1
+
+metadata:
+  label: fraction of military retirement benefits allowed as NE AGI subtraction
+  unit: /1
+  period: year
+  reference:
+  - title: 2022 NE income tax form and instruction booklet
+    href: https://revenue.nebraska.gov/sites/revenue.nebraska.gov/files/doc/2022_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_8.pdf#page=3
+  - title: Legislative Bill 387
```

Comment:
could we add the actual legal code section? - The section that is amended by this bill